File: app/src/main/java/com/paymob/weather/data/model/response/CityForecastResponse.java
package com.paymob.weather.data.model.response;

import java.util.List;
import com.google.gson.annotations.SerializedName;

public class CityForecastResponse{

	@SerializedName("city")
	private City city;

	@SerializedName("cnt")
	private int cnt;

	@SerializedName("cod")
	private String cod;

	@SerializedName("message")
	private int message;

	@SerializedName("list")
	private List<CityWeather> list;

	public void setCity(City city){
		this.city = city;
	}

	public City getCity(){
		return city;
	}

	public void setCnt(int cnt){
		this.cnt = cnt;
	}

	public int getCnt(){
		return cnt;
	}

	public void setCod(String cod){
		this.cod = cod;
	}

	public String getCod(){
		return cod;
	}

	public void setMessage(int message){
		this.message = message;
	}

	public int getMessage(){
		return message;
	}

	public void setList(List<CityWeather> list){
		this.list = list;
	}

	public List<CityWeather> getList(){
		return list;
	}
}

File: app/src/main/java/com/walid/weather/data/model/response/CityForecastResponse.kt
package com.walid.weather.data.model.response

import com.google.gson.annotations.SerializedName

data class CityForecastResponse(
    @SerializedName("city")
    var city: City? = null,

    @SerializedName("cnt")
    var cnt: Float = 0f,

    @SerializedName("cod")
    var cod: String? = null,

    @SerializedName("message")
    var message: Int = 0,

    @SerializedName("list")
    var list: List<CityWeather>? = null
)